main.tf

```
@@ -30,21 +30,28 @@ locals {
   })
 }
 
-resource "local_file" "vcsa_template_to_json" {
-  filename = "/tmp/vcsa/vcsa-${var.deploy_type}.json"
-  content  = local.vcsa_template
-}
+# resource "local_sensitive_file" "vcsa_template_to_json" {
+#   filename       = "/tmp/vcsa/vcsa-${var.deploy_type}.json"
+#   content        = local.vcsa_template
+# }
 
 resource "null_resource" "vcsa_linux_deploy" {
-  count = var.windows == false ? 1 : 0
+  count = var.linux == true ? 1 : 0
```

```suggestion
  count = var.linux ? 1 : 0
```

I wrote this before I was doing a ton of Terraform. Since `var.linux` is a type of `bool` we don't need have `== true` since `?` is checking if it's `true`.